android/app/src/main/kotlin/com/chicagoroboto/features/sessions/SessionDateView.kt

```diff
@@ -56,4 +59,14 @@ class SessionDateView(context: Context, attrs: AttributeSet? = null, defStyle: I
             tabs.visibility = View.VISIBLE
         }
     }
+
+    override fun scrollToCurrentDay() {
+        if (adapter.dates.isNotEmpty()) {
+            val format = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
```

Comment:
`Locale.US`? Someone's phone being set to Arabic won't change the need to parse regular numbers coming from the data source.